docs/manifest.md

```diff
@@ -99,9 +101,10 @@ opportunity tab extension.
   },
   "api": {
     "scopes": ["accounts.all", "calls.all"],
-    "applicationId": "AbCd123456qW",
-    "redirectUri": "https://application-host.com/hello-world",
-    "connect": "https://someurl.com/connect"
+    "client": {
```

Comment:
What is the reason we have here client: { id } vs having it just as clientId (renaming existing applicationId)?

Reply:
Just emphasizing that this is a separate entity and leaving a space for potential additions in a future. Would you prefer to have just `clientId` field?